state/allwatcher_internal_test.go

```diff
@@ -51,7 +54,7 @@ type allWatcherBaseSuite struct {
 // setUpScenario adds some entities to the state so that
 // we can check that they all get pulled in by
 // all(Model)WatcherStateBacking.GetAll.
-func (s *allWatcherBaseSuite) setUpScenario(c *gc.C, st *State, units int) (entities entityInfoSlice) {
+func (s *allWatcherBaseSuite) setUpScenario(c *gc.C, st *State, units int, perm permission.Access) (entities entityInfoSlice) {
```

Comment:
and here, rather than perm, change it to "setupApplicationOffer bool" or similar